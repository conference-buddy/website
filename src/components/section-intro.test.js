import React from "react";
import { screen, render, cleanup } from "@testing-library/react";
import IntroSection from "./section-intro";

describe("<IntroSection />", () => {
  let component;
  beforeAll(() => {
    component = render(<IntroSection />);
  });

  afterAll(cleanup);

  it("shows a header", () => {
    const header = screen.getByRole("heading", {
      name: "Conference Buddy",
    });

    expect(header).toBeVisible();
  });

  it("shows all elements", () => {
    expect(component.asFragment()).toMatchSnapshot();
  });
});
