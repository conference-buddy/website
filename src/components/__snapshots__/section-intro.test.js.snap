// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IntroSection /> shows all elements 1`] = `
<DocumentFragment>
  <div
    class="IntroSection IntroSection-fullscreen-bg bg-clip-ellipse"
  >
    <div
      class="container d-flex align-items-center p-content-section "
    >
      <div
        class="text-center text-light"
      >
        <h1
          class="IntroSection-logo-font"
        >
          <strong>
            Conference Buddy
          </strong>
        </h1>
        <div
          class="lead py-2 px-3"
        >
          <p>
            Conference Buddy is for everyone who wants to attend a tech conference but is 
            <strong>
              afraid to go alone
            </strong>
            . It's for everyone at tech events who is 
            <strong>
              not comfortable being on their own
            </strong>
             all the time. Itâ€™s for 
            <strong>
              all you
            </strong>
             first-timers, Introverts, socially awkwards, timid and shy people out there who are just like me!
          </p>
        </div>
      </div>
    </div>
    <div
      class="IntroSection-mouse-scroll-wrapper"
    >
      <div
        class="MouseScrollIcon d-none d-sm-block"
      >
        <a
          href="onPageLinkId-section-about"
        >
          <span
            class="MouseScrollIcon-mouse"
          >
            <span
              class="MouseScrollIcon-wheel"
            />
          </span>
        </a>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
